cmd/processManifest.go

```diff
@@ -12,20 +22,66 @@ var processManifestCmd = &cobra.Command{
 	Short: "Process a given manifest to meet the rhmi requirements.",
 	Long:  `Process a given manifest to meet the rhmi requirements.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("processImageManifests called")
+		//get csvfile
+		csv, filename, err := utils.ReadCSVFromBundleDirectory(manifestDir)
+		if err != nil {
+			handleError(err)
+		}
+		if filename == "" {
+			handleError(fmt.Errorf("No csv file found in the directory"))
+		}
+		fmt.Printf("Filename: %s", filename)
+		//populate a csv object from the file
+		filepath := fmt.Sprintf("%s/%s", manifestDir, filename)
+		fmt.Printf("Filepath: %s", filepath)
+		err = utils.PopulateObjectFromYAML(filepath, csv)
+		if err != nil {
+			handleError(err)
+		}
```

Comment:
Is any of this required? ReadCSVFromBundleDirectory shoudl return a csv object

Reply:
Yep, the filepath is needed so both are needed.